app/components/base/PageFeedback.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Button, styled, Typography } from "@mui/material";
+import {
+  type SvgIconComponent,
+  ThumbDownOffAltOutlined,
+  ThumbUpOffAltOutlined,
+} from "@mui/icons-material";
+import merge from "lodash.merge";
+import { Link } from "./Typography";
+
+const FeedbackButton = styled(Button, { name: "FeedbackButton" })(() => ({
+  color: "rgba(0, 0, 0, 0.50)",
+  textTransform: "none",
+}));
+
+const FeedbackContainer = styled(Box, { name: "FeedbackContainer" })(() => ({
+  alignItems: "center",
+  display: "flex",
+  flexDirection: "row",
+  gap: "12px",
+}));
+
+const FeedbackInnerText = styled(Typography, { name: "FeedbackInnerText" })(
+  () => ({
+    color: "rgba(106, 114, 130, 1)",
```

Comment:
Same for this one: could we add this as a variable to the `app/components/colors.ts` file?

Not sure what the naming should be for this one though 😅 😁